index.js

```diff
@@ -1,11 +1,12 @@
+const jsesc = require('jsesc');
+
 module.exports = function (source) {
   if (this.cacheable) this.cacheable();
 
   var value = typeof source === "string" ? JSON.parse(source) : source;
 
-  value = JSON.stringify(value)
-    .replace(/\u2028/g, '\\u2028')
-    .replace(/\u2029/g, '\\u2029');
+  value = JSON.stringify(value);
+  value = jsesc(value);
```

Comment:
Is `jsesc(value, { json: true })` a thing/require/needed here? (I'm not familiar with this module)

Reply:
See https://github.com/webpack-contrib/raw-loader/pull/36#discussion_r141774772 for reference

Reply:
Given webpack handles json at this point, why are we updating this here?

Reply:
'Native JSON Support' turned out to be mainly a  `push(jsonLoader)` for `.json` if missing https://github.com/webpack/webpack/pull/3374/files#diff-6cff30a1f2b17edb1b17434da5a7d140

Reply:
lol, of course